src/Microsoft.NetCore.Analyzers/Microsoft.NetCore.Analyzers.sarif

```diff
@@ -325,6 +325,25 @@
             ]
           }
         },
+        "CA2012": {
+          "id": "CA2012",
+          "shortDescription": "Use ValueTasks correctly",
+          "fullDescription": "ValueTasks must only be consumed once, generally by directly awaiting them.  Attempts to consume a ValueTask multiple times may result in an exception or corruption.",
+          "defaultLevel": "warning",
```

Comment:
Was this file generated prior to changing the `RuleLevel` values in the analyzer? `"defaultLevel": "warning"` indicates `RuleLevel.BuildWarning` was used when creating the analyzer descriptors.

I am wondering if this rule should be a `RuleLevel.BuildWarningCandidate`, that should be triaged to be enabled as a warning by default?

Reply:
Actually, I take my comment back. I did not see that this was `Microsoft.NetCore.Analyzers.sarif`, which is the analyzer assembly that ships with FxCopAnalyzers package, where we do want all analyzers to be warnings by default. I do see further down that `Microsoft.CodeAnalyzers.NetAnalyzers.sarif` does have this rule listed as a hidden rule by default.